recipes/mariadb-connector-c/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mariadb-connector-c" %}
+{% set version = "3.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mariadb-corporation/mariadb-connector-c/archive/v{{ version }}.tar.gz
+  sha256: c0fda1fa6e52dc85de27156cd847088a72d40d9de6514f7efa57c8d93134a54c
+
+build:
+  number: 0
+  ignore_run_exports:
+      - libcxx  # [osx]
```

Comment:
I'm not sure about this part. @isuruf do we need this here?

Reply:
@ocefpaf should I remove this from the meta.yaml file?

Reply:
Let's try and test if you really need it.

Reply:
I updated the meta file and it looks like we did not need it for osx